shapeshift/web
# feat: adding chinese translations (zh) and language option

## Description

- Adds the Chinese language in the UI
- Adds Simplified Chinese translations (zh)



## Pull Request Type


## Risk
Low. New option in the UI. Any missing translation (there should be none) will fallback to English.



## Testing

Please allocate some time for our Chinese translator to browse/use the app while the feature is available either on develop or release so they can review the translations in place. They have done it based on few screenshots, but it would be good that they interact with the app in their language before release.

- Check that Chinese (中文) is available and working in the Language Selector (on the welcome screen) and in the Settings modal
- Check if the UI is broken anywhere in the app due to the new strings.
- Check that the Chinese language is detected if it's the default language in the browser during the first visit (you can use the Clear Cache feature for this now as it clears this setting too).
- All the strings have been translated at the time I'm writing this, so any untranslated string is likely to be hardcoded. I know [some part of the Swapper UI needs some work](https://github.com/shapeshift/web/issues/4733) in this regard (e.g. "Balance" in one the screenshots below), you can disregard these, they will be translated once it is possible.



### Engineering
☝️ 


### Operations
☝️ 


## Screenshots (if applicable)
![image](https://github.com/shapeshift/web/assets/88804546/fc5dbfc3-7b9c-4861-b2f8-cc2f8295f6eb)

![image](https://github.com/shapeshift/web/assets/88804546/784115a8-37ca-4981-8671-f55fb665f36d)

![image](https://github.com/shapeshift/web/assets/88804546/6db51f14-0eae-40fb-b49b-aacf4513bcaf)

![image](https://github.com/shapeshift/web/assets/88804546/eb1d6da5-8f49-4c98-a844-aa52fa84d723)

![image](https://github.com/shapeshift/web/assets/88804546/12d737a6-3d1a-4ec7-83bc-faf5aa1d50b8)

![image](https://github.com/shapeshift/web/assets/88804546/be2517b3-718d-4002-87e7-99ca2fc0f206)
